packages/components/scripts/generateDSRComponents.ts

```diff
@@ -28,6 +28,7 @@ const generateDSRComponents = (): void => {
 
       let newFileContent = fileContent
         .replace(/@Component\({[\s\S]+?\)\n/g, '')
+        .replace(/\simplements\s\w+/g, '')
```

Comment:
This would be more precise and probably easier to read, too.
```suggestion
        .replace(/ implements [A-Za-z]+/g, '')
```